cpp/src/io/utilities/hostdevice_vector.hpp

```diff
@@ -162,7 +162,7 @@ class hostdevice_vector {
     v.h_data       = nullptr;
   }
 
-  rmm::cuda_stream_view stream{cudf::default_stream_value};
+  rmm::cuda_stream_view stream{cudf::default_stream_value};  // TODO: remove stream member?
```

Comment:
This is resolved by #11631.
```suggestion
  rmm::cuda_stream_view stream{cudf::default_stream_value};
```